code/main/edge/edgeRequestWorker.brs

```diff
@@ -59,8 +65,9 @@ function _adb_EdgeRequestWorker() as object
             responseArray = []
             while m.hasQueuedEvent()
 
-                if _adb_isEmptyOrInvalidString(ecid) or _adb_isEmptyOrInvalidString(configId) then
-                    _adb_logWarning("processRequests() - Edge request skipped. ECID and/or configId not set.")
+                currTS = _adb_timestampInMillis()
+                if (m._lastFailedRequestTS <> m._INVALID_WAIT_TIME) and ((currTS - m._lastFailedRequestTS) < m._RETRY_WAIT_TIME_MS)
```

Comment:
Optional: we can eliminate the use of `_INVALID_WAIT_TIME` and introduce a new private function, `_resetLatFailedRequestTS()`, which will update `m._lastFailedRequestTS` to any negative value and do `if (m._lastFailedRequestTS > 0 ) and (...)` here.

Reply:
I would prefer to keep the current logic, if you are okay with it.